src/models/zone-group.ts

```diff
@@ -15,6 +15,7 @@ export interface ZoneMember {
   uuid: string;
   name: string;
   ChannelMapSet?: ChannelMapSet;
+  Satellites?: ZoneMember[];
```

Comment:
In addition to supporting home theater (ht) channel map sets, this will also pull in ht satellites.

It will not recurse beyond 1 level, as satellites do not/can not have their own satellites.